test_runner/fixtures/zenith_fixtures.py

```diff
@@ -128,15 +129,29 @@ class Postgres:
     def __init__(self, zenith_cli):
         self.zenith_cli = zenith_cli
         self.running = False
+        self.host = 'localhost'
+        self.port = 55432
         # path to conf is <repo_dir>/pgdatadirs/pg1/postgresql.conf
 
-    def create_start(self):
+    def create_start(self, config_lines=[]):
```

Comment:
An empty list as a default parameter [can create problems](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument) in python.

The recommended fix is:
```
def create_start(self, config_lines=None):
    if config_lines = None:
        config_lines = []
```